### PR TITLE
empty commit

### PR DESCRIPTION
**Pull request by IkmosaO**
### goal of the PR
The goal of this PR is to set a basic route with a skeleton.

### What changes were introduced, and the expected behavior
I have created the routes for the customer, and I have also created the skeletons for the controllers.
The expected behavior is to be able to post through postman to the customer routes, and not see any errors.

### CODE REVIEW

**Comment by IkmosaO** [comment]
Please check out my index.js, added a few things to file. What do you guys think?
@grimmi5894 @catb0591 @JesusOrozco978 @IgnacioOAF

File: controller/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+}
+
+module.exports = {
+    createNewPerson,
+    getAllContacts,
+    getContactById,
+    uploadContacts,
+    uploadCSVFile
+}
```
Comment (IgnacioOAF) [inline controller/index.js]: Nit: here you need to add an empty line.

**Review by IgnacioOAF — APPROVED** [review]
Great job here

**Review by catb0591 — COMMENTED** [review]
Going to look at this!

**Review by IgnacioOAF — CHANGES_REQUESTED** [review]
change this

**Review by IgnacioOAF — APPROVED** [review]

**Review by catb0591 — APPROVED** [review]
This looks great @IkmosaO 
Let me know when this is merged so I can pull this up and start tackling some of the controllers :)

**Comment by catb0591** [comment]
@IgnacioOAF - are we okay to merge this when @IkmosaO adds the space at the end?

**Comment by Ignacio01** [comment]
Yes, this should be safe to merge.

**Comment by IkmosaO** [comment]
Alright i added that space, fixed a spelling error in the controllers/index.js  on line 27, and did some formatting.

I'll be merging this branch to main now.

**Pull request merged**